cruiser_test.go

```diff
@@ -13,7 +13,7 @@ func TestCruiser_RapidfireAgainst(t *testing.T) {
 
 func TestCruiser_GetCargoCapacity(t *testing.T) {
 	c := newCruiser()
-	assert.Equal(t, 800, c.GetCargoCapacity())
+	assert.Equal(t, 928, c.GetCargoCapacity(Researches{HyperspaceTechnology: 8}))
```

Comment:
I would keep the base test (tech: 0 = 800cargo) and make a new test for (tech: 8 = 928)